python/ribasim_testmodels/ribasim_testmodels/basic.py

```diff
@@ -336,8 +336,7 @@ def tabulated_rating_curve_model() -> ribasim.Model:
             data={
                 "node_id": [3, 3, 3, 3, 3, 3],
                 "time": [
-                    # test subsecond precision
-                    pd.Timestamp("2020-01-01 00:00:00.000001"),
+                    pd.Timestamp("2020-01-01 00:00:01"),
```

Comment:
Unfortunately subsecond precision wasn't working at all. See #1095 for a separate PR.